docs/en/latest/USER_GUIDE.md

```diff
@@ -23,31 +23,31 @@ title: User Guide
 
 The following are parts of the modules' snapshot.
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/113472303-f156cb80-9494-11eb-8416-dddb765854bb.png)

Hi, I noticed there has an old image 🤔 we don't have plugin form currently.

Reply:
Will fix it asap

Reply:
got it